packages/astro/test/content-collections-render.test.js

```diff
@@ -195,5 +212,15 @@ describe('Content Collections - render()', () => {
 			expect(h2).to.have.a.lengthOf(1);
 			expect(h2.attr('data-components-export-applied')).to.equal('true');
 		});
+
+		it('Renders with raw html for markdown content', async () => {
+			const response = await fixture.fetch('/render-with-html', { method: 'GET' });
+			expect(response.status).to.equal(200);
+
+			const html = await response.text();
+			const $ = cheerio.load(html);
+
+			expect($('p').text()).to.equal('This is a post');
+		});
 	});
 });
```

Comment:
nit: as a sanity check, I might add a unit test from a `.ts` file as well. Just to confirm `html` is accessible when generating RSS feeds.

Reply:
I'm not exactly sure what you mean about "unit test from a `.ts` file", but I'm guessing creating a `.ts` endpoint and set up an integration test for it? Not sure how a unit test would work with a `.ts` file with our current setup 😅

Reply:
@bluwy Yes, exactly. I've set up some JSON endpoint unit tests before, and they're honestly simpler than our HTML tests. [Example here.](https://github.com/withastro/astro/blob/main/packages/astro/test/content-collections.test.js#L18-L19) This uses the devalue package to parse not-JSON-friendly properties, but you could use `JSON.parse()` for these tests instead.

Reply:
83b0815 should do it. Thanks for the tip!